src/chainparams.cpp

```diff
@@ -280,7 +276,7 @@ class CTestNetParams : public CChainParams {
  */
 class CRegTestParams : public CChainParams {
 public:
-    CRegTestParams() {
+    CRegTestParams(ArgsManager& args) {
```

Comment:
Is it necessary to pass in an entire ArgsManager object? It seems only the -vbparams value is used. What about just passing in the vbparams value?

Reply:
Well, the point of that is to make it easy to add further params, like in https://github.com/bitcoin/bitcoin/pull/8994 (probably not all of them are interesting for testing).

Reply:
Could make this `const ArgsManager& args`